docs/development/virtual_environments.rst

```diff
@@ -45,6 +45,7 @@ To get started working with the development environment:
    ./manage.py run         # run development servers
    ./manage.py reset       # resets the state of the development instance
    ./manage.py add-admin   # create a user to use when logging in to the Journalist Interface
+   ./manage.py translate-messages --compile   # add languages and display the i18n menu
```

Comment:
I think we'll just need to add this in the Dockerfile, since in #2695 I added this task in the Ansible development VM.

But for the language picker to be displayed, we'll need to add an Ansible task / RUN command that adds locales to `SUPPORTED_LOCALES` in `config.py` (probably only worth doing for Docker since the Ansible based development environment is slated for removal in 0.6).

Reply:
Oh, right. I'll revise this PR accordingly, I was too quick. Thanks !